### PR TITLE
Yegna Templates setup

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

File: gem/templates/yegna/base.html [hunk 1]
```diff
@@ -148,10 +150,12 @@
               </div>
             </li>
             {% comment %}
+
```
Comment (sewagodimo) [inline gem/templates/yegna/base.html]:
```suggestion
```

File: gem/templates/yegna/base.html [hunk 2]
```diff
@@ -121,7 +121,9 @@
           {% endif %}
         </a>
       </div>
+
       {% load_sections as sections %}
+
```
Comment (sewagodimo) [inline gem/templates/yegna/base.html]:
```suggestion
```

**Comment by codiebeulaine** [comment]
👍

File: gem/templates/yegna/base.html [hunk 3]
```diff
@@ -148,10 +150,12 @@
               </div>
             </li>
             {% comment %}
+            
               <!-- WHAT WERE THE DESIGNER'S INTENTIONS&ASSUMPTIONS HERE -->
               <!-- STATE HIGHTLIGHT: Active section -->
             {% endcomment %}
             {% for section_page in sections %}
+            {{section_page}}
```
Comment (codiebeulaine) [inline gem/templates/yegna/base.html]: @Mitso should this still be here?

**Pull request merged**